defaults/main.yml

```
@@ -81,6 +81,10 @@ odoo_role_enable_dbfilter_from_header: false
 odoo_role_limit_time_cpu: 60
 odoo_role_limit_time_real: 120
 
+# Customize the Odoo memory limits
+odoo_role_limit_memory_hard : 2684354560
```

The default value in Odoo is: 2048 * 1024 * 1024 = 2147483648

https://git.coopdevs.org/coopdevs/odoo/OCB/-/blob/12.0/odoo/tools/config.py#L303

```suggestion
odoo_role_limit_memory_hard : 2147483648
```